cylc/flow/rundb.py

```diff
@@ -706,7 +706,7 @@ def select_task_job_platforms(self):
     def select_submit_nums(self, name, point):
         """Select submit_num and flow_nums from task_states table.
 
-        Fetch submit number and flow_nums for spawning task name.point.
+        Fetch submit number and flow_nums for spawning task point/name.
```

Comment:
This could potentially be confusing
```suggestion
        Fetch submit number and flow_nums for spawning the task
        with the given name and cycle point.
```